pkgs/by-name/di/display3d/package.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "display3d";
+  version = "0.1.17";
+
+  src = fetchCrate {
```

Comment:
I don't know if there is anything wrong with `fetchCrate`, so it is your choice, but normally when the source is on GitHub, the source would be fetched from there with `fetchFromGithub`